staging/src/k8s.io/kubectl/pkg/cmd/debug/profiles.go

```diff
@@ -261,9 +261,11 @@ func dropCapabilities(p *corev1.Pod, containerName string) {
 		if c.SecurityContext == nil {
 			c.SecurityContext = &corev1.SecurityContext{}
 		}
-		c.SecurityContext.Capabilities = &corev1.Capabilities{
-			Drop: []corev1.Capability{"ALL"},
+		if c.SecurityContext.Capabilities == nil {
```

Comment:
Wouldn't it be better;

```go
c.SecurityContext.Capabilities = &corev1.Capabilities{
			Drop: []corev1.Capability{"ALL"},
                        Add: nil,
		}
```

Reply:
The updated style allows this to be consistent with the approach in `setPrivileged` and ensures that we don't inadvertently unset  other unrelated SecurityContext struct fields.

Reply:
Although `Capabilities` contains only 2 fields now, it might change in the future. To prevent possible unrelated overrides, You are right.